src/components/Posts/components/Post.tsx

```diff
@@ -0,0 +1,47 @@
+import { HeartOutlined } from '@ant-design/icons';
+import React from 'react';
+
+import {
+  PostContainer,
+  PostTitle,
+  PostSubtitle,
+  PostBody,
+  Author,
+  CommentCount,
+  LikeButton,
+  BookmarkButton,
+  PostHeader,
+  PostFooter,
+} from './styled';
+import { IPostComponent } from './types';
+
+const Post: React.FC<IPostComponent> = ({
```

Comment:
```suggestion
const Post: FC<IPostComponent> = ({
```
Fix please among all the app